ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -117,6 +117,8 @@ type ssh_daemon_type = {
     "options" ? ssh_daemon_options_type
     "comment_options" ? ssh_daemon_options_type
     "sshd_path" ? string
+    # if true and sshd doesn't exist, will skip config validation
```

Comment:
Can you add this to the pod too please? I think "validation_optional" would be a more correct name too? Or we could reverse the sense and say ```"always_validate" : boolean = true``` ?